etcdserver/server_test.go

```diff
@@ -451,6 +451,7 @@ func TestApplyConfChangeError(t *testing.T) {
 		srv := &EtcdServer{
 			r:       raftNode{Node: n},
 			cluster: cl,
+			cfg:     &ServerConfig{},
```

Comment:
I would like to follow the field order in the struct: https://github.com/coreos/etcd/blob/master/etcdserver/server.go#L144
So the best place should be the second line actually. (I remembered the wrong order when i left the comment)